blockchain/state_migration.go

```diff
@@ -100,12 +100,12 @@ func (bc *BlockChain) migrateState(rootHash common.Hash) error {
 
 	start := time.Now()
 
-	srcCachedDB := bc.StateCache().TrieDB()
-	targetDB := statedb.NewDatabase(&stateTrieMigrationDB{bc.db})
+	srcState := bc.StateCache()
+	targetState := state.NewDatabase(&stateTrieMigrationDB{bc.db})
```

Comment:
The word `target` seems vague. Both old DB and new DB can be target in migration. Can you change to `dstState` like  in the test cases?
```suggestion
	dstState := state.NewDatabase(&stateTrieMigrationDB{bc.db})
```

Reply:
reflected. :)